plugins/woocommerce/includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -619,7 +619,7 @@ private static function _update_check( $payload ) {
 
 		$cache_key = '_woocommerce_helper_updates';
 		$data      = get_transient( $cache_key );
-		if ( false !== $data ) {
+		if ( false !== $data && is_array( $data ) ) {
 			if ( hash_equals( $hash, $data['hash'] ) ) {
```

Comment:
For an abundance of caution, should we also check if `$data['products']` is an array?